src/api/Web.Bff.Shopping/Config/ProductApiOperations.cs

```diff
@@ -10,5 +10,6 @@ public class ProductApiOperations
             public static string DeleteProduct(int id) => $"api/products/{id}";
             public static string EditProduct => $"api/products";
         }
+        
```

Comment:
Next time, try to avoid changes like these in files - they're littering the file history, making its harder to find something specific.